plugin/build.gradle.kts

```diff
@@ -57,12 +58,14 @@ shadowJar.configure {
     }
 }
 
-tasks.getByName("jar").enabled = false
-tasks.getByName("jar").dependsOn(shadowJar)
+tasks.getByName("jar").apply {
+    enabled = false
+    dependsOn(shadowJar)
+}
 
 gradlePlugin {
     plugins {
-        create("testRetry") {
+        register("testRetry") {
             id = "org.gradle.test-retry"
             displayName = "Gradle test retry plugin"
             description = project.description
```

Comment:
my understanding is that `implementationClass` makes the `plugin/src/main/resources/META-INF/gradle-plugins/org.gradle.test-retry.properties` redundant, hence the duplicate warn - please validate

Reply:
Indeed, the `java-gradle-plugin` does generate that file. Good catch! 👍